birdhouse/optional-components/weaver/config/weaver/wps_processes.yml.template

```diff
@@ -0,0 +1,27 @@
+# Details:
+#   https://github.com/crim-ca/weaver/blob/master/config/wps_processes.yml.example
+#
+# For each of the below WPS service providers, Weaver will iterate over listed WPS processes returned by GetCapabilities
+# and will register them as individual WPS-REST processes. Each process will be referenced
+
+# using 'providers' instead of 'processes' will register services directly without fetching each process (local copy)
+# this will make resulting deployed processes dynamic according to what those providers actually offer at any given time
+providers:
```

Comment:
Does weaver have a "service user" with some privileges that would allow him to access the following routes if they are not public or do we need to validate that every provider is public?

Reply:
For the moment, all providers are anonymous-read recursively since the processes are accessible anyway via their respective WPS-1 endpoints. Technically, the POST of jobs could also be anonymous-accessible since it is possible to run the process execution via the WPS-1 GET requests of each provider real location.

Local processes (eg: deployed docker) do actually require read/write permissions to be added. For example, https://132.217.140.4/twitcher/ows/proxy/weaver/processes/jsonarray2netcdf fails if you are not logged in as Magpie admin.

Reply:
Ok, but my point is only for static providers. To register their processes weaver will need to acces them while booting. What if they are not public? Maybe add a note in the file saying that weaver fetches processes anonymously so they must be public.

Reply:
Yes they will fail if they are not publicly accessible.